sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2995,6 +3260,26 @@ def hpat_pandas_series_nunique_impl(self, dropna=True):
 @overload_method(SeriesType, 'count')
 def hpat_pandas_series_count(self, level=None):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.count
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/count.py
+       :language: python
+       :lines: 27-
+       :caption: Getting the number of not NaN values in Series
```

Comment:
Consider alternative caption: Counting non-Nan values in series. 
Or alternatively use the caption suggested in values_count()